template/package.json.j2

```diff
@@ -0,0 +1,78 @@
+        "title": "{%- if meta.versions[v].title %}{{ meta.versions[v].title }}{%- endif %}",
+        "description": "{%- if meta.versions[v].description %}{{ meta.versions[v].description }}{%- endif %}",
+        "license": "{%- if meta.versions[v].license %}{{ meta.versions[v].license }}{%- endif %}",
+        "files": [
+          {%- if meta.versions[v].files %}
+          {% for f in meta.versions[v].files -%}
+          "{{ f }}"{%- if not loop.last %},{%- endif %}
+          {% endfor %}
+          {%- endif %}],
+        "main": "{%- if meta.versions[v].main %}{{ meta.versions[v].main }}{%- endif %}",
+        "url": "{%- if meta.versions[v].url %}{{ meta.versions[v].url }}{%- endif %}",
+        "homepage": "{%- if meta.versions[v].homepage %}{{ meta.versions[v].homepage }}{%- endif %}",
+        "author": "{%- if meta.versions[v].author %}{{ meta.versions[v].author }}{%- endif %}",
+        "repository": {
+          {%- if meta.versions[v].repository %}
+          {% for r in meta.versions[v].repository -%}
+          "{{ r }}": "{{ meta.versions[v].repository[r] }}"{%- if not loop.last %},{%- endif %}
+          {% endfor %}
+          {%- endif %}},
+        "dependencies": {
+          {%- if meta.versions[v].dependencies %}
+          {% for d in meta.versions[v].dependencies -%}
+          "{{ d }}": "{{ meta.versions[v].dependencies[d] }}"{%- if not loop.last %},{%- endif %}
+          {% endfor %}
+          {%- endif %}},
+        "scripts": {
+          {%- if meta.versions[v].scripts %}
+          {% for s in meta.versions[v].scripts -%}
+          "{{ s }}": "{{ meta.versions[v].scripts[s] }}"{%- if not loop.last %},{%- endif %}
+          {% endfor %}
+          {%- endif %}},
+        "babel": {
+          {%- if meta.versions[v].babel %}
+          {% for b in meta.versions[v].babel -%}
+          "{{ b }}": "{{ meta.versions[v].babel[b] }}"{%- if not loop.last %},{%- endif %}
+          {% endfor %}
+          {%- endif %}}
+      }{%- if not loop.last %},{%- endif %}
+    {% endfor %}
+  {%- endif %}},
+  "author": "{%- if meta.author is defined %}{{ meta.author }}{%- endif %}",
+  "license": "{%- if meta.license is defined %}{{ meta.license }}{%- endif %}",
+  "repository": {
+  {%- if meta.repository is defined %}
+    {% for repo in meta.repository -%}
+      "{{ repo }}": "{{ meta.repository[repo] }}"{%- if not loop.last %},{%- endif %}
+    {% endfor %}
+  {%- endif %}},
+  "maintainers": [
+  {%- if meta.maintainers is defined %}
+    {% for m in meta.maintainers -%}
+      "{{ m }}"{%- if not loop.last %},{%- endif %}
+    {% endfor %}
+  {%- endif %}],
+  "keywords": [
+  {%- if meta.keywords is defined %}
+    {% for key in meta.keywords -%}
+      "{{ key }}"{%- if not loop.last %},{%- endif %}
+    {% endfor %}
+  {%- endif %}],
+  "bugs": "{%- if meta.bugs is defined %}{{ meta.bugs }}{%- endif %}",
+  "homepage": "{%- if meta.homepage is defined %}{{ meta.homepage }}{%- endif %}"
+}
```

Comment:
As python has native ability to generate json file, I'm thinking if this template is really needed. If we use the jinja template to generate package.json, we need to check the empty values with keys. For example, the last "homepage":"...", if the meta.homepage is not defined, I think the key "homepage" should also not be there. 
So @yma88  can you check if it is easy to generate the json file through the class object directly and if it is easy to do? If yes, we can do it instead and avoid this template I think